exporter/queue/listener.py

```diff
@@ -38,6 +38,6 @@ def try_handle_or_reject(self, body: str, msg: Message):
             try:
                 self.handler.handle_message(json_body, msg)
             except Exception as e:
-                s.logger.error(f"Rejecting message: {body} due to error: {str(e.__class__.__name__)}")
+                s.logger.error(f"Rejecting message: {body} due to error: {str(e) if str(e) else e.__class__.__name__}")
```

Comment:
I see you use this construct in at least another location. I would say this merits a function.

Reply:
happy to introduce this later